src/Components/sharedComponents/BetaLabel.scss

```diff
@@ -0,0 +1,3 @@
+.beta-label {
+    --pf-c-label--BackgroundColor: var(--pf-global--palette--green-300);
```

Comment:
Awesome! One more tiny nitpick, can we add:
`--pf-c-label__content--before--BorderColor: var(--pf-global--palette--green-300);`
to get rid of the grey border? It is barely perceptible, but adds a weird haze around the button (at least on my low dpi monitor...).

Reply:
of course